src/platform/api.py

```diff
@@ -108,3 +111,41 @@ def get_variables_and_secrets() -> list[dict]:
     response = _make_get_request(f"/computebenches/{_get_org_id()}/placeholder/secrets")
 
     return response["data"]
+
+
+@functools.cache
+@beartype
+def get_public_keys() -> list[dict]:
+    """get public keys from public endpoint"""
+
+    jwks_url = urljoin(get_value()["auth_domain"], ".well-known/jwks.json")
+    data = requests.get(jwks_url).json()
+    return data["keys"]
+
+
+@beartype
+def decode_access_token() -> dict:
+    """decode access token into human readable data"""
+
+    tokens = auth.get_tokens()
```

Comment:
Ditto. Idea being -- imagine writing tests for these functions, as is. You'll have to mock `auth` and `get_value`. Alternatively, if you had pure versions of these functions, you could just pass values directly in a test.